pywinauto/controls/uiawrapper.py

```diff
@@ -490,6 +522,13 @@ def _select(self, item = None):
         else:
             raise IndexError("item not found")
 
+    #-----------------------------------------------------------
+    def is_active(self):
+        """Whether the window is active or not"""
+        minimized = self.iface_window.CurrentWindowVisualState == \
+                       uia_defs.window_visual_state_minimized
+        return (not minimized and self.is_visible() and self.is_enabled())
```

Comment:
As an initial implementation it solves exception #166. But how about the case when the app window is normal, enabled and visible but some other app opens a window on top of the our app? We can negatively test it with 2 instances of any MFC sample.